spec/support/examples/api.rb

```diff
@@ -21,3 +21,35 @@
     expect(response).to have_http_status(403)
   end
 end
+
+shared_examples 'unprocessable entity' do
+  it 'returns http unprocessable entity' do
+    # Some examples have a subject which needs to be called to make a request
+    subject if request.nil?
+
+    expect(response).to have_http_status(422)
+  end
+end
+
+shared_examples 'unauthorized for invalid token' do
+  context 'with empty Authorization header' do
+    let(:headers) { { 'Authorization' => '' } }
+
+    it 'returns http unauthorized' do
+      # Some examples have a subject which needs to be called to make a request
+      subject if request.nil?
+
+      expect(response).to have_http_status(401)
+    end
+  end
+
+  context 'without Authorization header' do
+    let(:headers) { {} }
```

Comment:
This might be more semantically correct, as to not mess with Accept or other headers that might be being used for the request:
```suggestion
    let(:headers) { headers.delete('Authorization') }
```